### ISSUE TITLE
Versions & restore

### ISSUE THREAD

**Issue by camilasan** [nextcloud/desktop#1140]
cc @jancborchardt

**Comment by jancborchardt** [comment]
Yep, so basically this should be another tab in the dialog which is currently for Sharing only. It should be modeled after the web interface. Having tabs for Activity (Activity, Comments and Versions) and Sharing.

**Comment by BloodyIron** [comment]
#1102 supersedes this, please close this one as duplicate and re-open the original issue, and work there. We still need this.

**Comment by joshtrichards** [comment]
I believe there are two parts to versioning support for the Desktop client:

1. Exposing access to the versions already managed by the server (basically a UI matter).
2. Whatever is deemed possible/ desirable to support in terms of use cases when files are modified via the Desktop client.

#1102 does a good job describing some of the latter in particular. I reopened it to cover that for the time being since none of that is covered here in this issue.

**Comment by Rello** [comment]
closing this one in favor of #1102
